src/components/home/elements/DepositToken.vue

```diff
@@ -626,15 +494,23 @@ export default {
         checkPrecision(amount) {
             return common.checkPrecision(BigNumber(amount).multipliedBy(this.tokenUnity), 0)
         },
-        isExceededMaxSupply(amount) {
+        isExceededBalance(amount) {
             return BigNumber(amount).isGreaterThan(this.tokenBalance)
         },
         isInsufficient() {
             return BigNumber(this.balance).isLessThan(BigNumber(CONTRACT_EXEC_FEE))
         },
         isValidContractId(contractId) {
-            var contractIdCmp = transaction.tokenIDToContractID(this.tokenId)
-            return contractId === contractIdCmp
+            var contractArr = base58.decode(contractId)
+            if (contractArr) {
```

Comment:
simplify this code to `return contractArr && contractArr.length === 26 && contractArr[0] === 6`